BUILD.bazel

```diff
@@ -1,5 +1,8 @@
 load("@rules_proto//proto:defs.bzl", "proto_library")
-load("@rules_python//python:defs.bzl", "py_binary")
+
+# XXX in bzlmod this works, but in workspace we need
```

Comment:
The XXX comment should be replaced with something like this
```suggestion
# NOTE: if you're still using WORKSPACE, you'll need to patch this file to use
# the following instead
```

Reply:
Done